docs/tutorials/kubernetes-cluster-in-aws-with-gardener/_index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Create a kubernetes cluster in AWS with Gardener
+level: beginner
+category: Getting Started
+reviewer:
+status:
+last_reviewed:
+scope: app-developer
+---
+
+### Prerequisites
+
+-   You need an AWS account.
+-   You have access to the Gardener dashboard and have permissions to create projects.
+
+1. Go to the Gardener dashboard and create a *Project*.
+
+    <img src="images/new_gardener_project.jpg">
+
+
+2. To copy the policy for AWS from the Gardener dashboard, choose *Secrets*, click on the help icon <img src="images/help_icon.jpg"> for AWS secrets, and choose copy <img src="images/copy_icon.png">.
```

Comment:
I think the `<img src="images/help_icon.jpg">` part is a broken link, the file name currently is `help_icon.JPG`. Can you rename the file to `help_icon.jpg` to fix the broken link and to make image extensions consistent (all lowercase)?